diff --git a/node_modules/web3-utils/lib/index.js b/node_modules/web3-utils/lib/index.js
index 09ebd7f..7e64e39 100644
--- a/node_modules/web3-utils/lib/index.js
+++ b/node_modules/web3-utils/lib/index.js
@@ -260,8 +260,8 @@ var toWei = function (number, unit) {
 var toChecksumAddress = function (address) {
     if (typeof address === 'undefined')
         return '';
-    if (!/^(0x)?[0-9a-f]{40}$/i.test(address))
-        throw new Error('Given address "' + address + '" is not a valid Ethereum address.');
+    // if (!/^(0x)?[0-9a-f]{40}$/i.test(address))
+    //     throw new Error('Given address "' + address + '" is not a valid Ethereum address.');
     address = address.toLowerCase().replace(/^0x/i, '');
     var addressHash = utils.sha3(address).replace(/^0x/i, '');
     var checksumAddress = '0x';
